README.md

```diff
@@ -10,13 +22,87 @@ source code base for general changes.
 
 This repo does not include all assets necessary for compiling the ROM. A prior copy of the game is required to extract the required assets.
 
-This is a decompilation of Legend of Zelda: Majora's Mask (U) 1.0
+This is a decompilation of Legend of Zelda: Majora's Mask (US) 1.0
 
 It builds the following ROM:
 * rom.z64 `md5: 2a0a8acb61538235bc1094d297fb6556`
 
-Please refer to the Installation guide in the Wiki for setup instructions.
+Please refer to the following for more information:
 
 Website: https://zelda64.dev/
 
 Discord: https://discord.zelda64.dev
+
+## Installation
+
+### Windows
+
+For Windows 10, install WSL and a distribution by following this
+[Windows Subsystem for Linux Installation Guide](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+We recommend using Debian or Ubuntu 18.04 Linux distributions.
+
+### Linux (Native or under WSL / VM)
+
+#### 1. Install build dependencies
+
+The build process has the following package requirements:
+
+* make
+* git
+* build-essential
+* binutils-mips-linux-gnu
+* python3
+* pip3
+
+Under Debian / Ubuntu (which we recommend using), you can install them with the following commands:
+
+```bash
+sudo apt update
+sudo apt install make git build-essential binutils-mips-linux-gnu python3 python3-pip
+```
+
+#### 2. Fork the repository
+
+Create your own fork of the repository at `https://github.com/zeldaret/mm`. Then clone your fork where you wish to have the project, with the command:
+
+```bash
+git clone https://github.com/<YOUR_USERNAME>/mm.git
+```
+
+#### 3. Prepare a base ROM
+
+Copy your ROM to inside the root of this new project directory, and rename the file to "baserom.z64".
```

Comment:
This should probably accompany a rename of the baserom to reflect the version. The naming format we will be using in the future can be found below. We also want to make sure the Jenkinsfile is updated with the new rom destination as well.
https://github.com/zeldaret/mm/blob/e97f10a6fa573e8b817ed03a4674d52b1818dafa/Jenkinsfile#L10

Reply:
```suggestion
Copy your ROM to inside the root of this new project directory, and rename the file of the baserom to reflect the version of ROM you are using. ex: `mm.us.rev1.z64`
```
So a change to something like this?

Reply:
Yes.

Reply:
I forgot one more line that needs to change:
https://github.com/zeldaret/mm/blob/e97f10a6fa573e8b817ed03a4674d52b1818dafa/Makefile#L69